lib/_components/appGithubDiscussion/template.njk

```diff
@@ -0,0 +1,12 @@
+{% from 'input/macro.njk' import input %}
+{% from 'textarea/macro.njk' import textarea %}
+{% from "button/macro.njk" import button %}
+{% from 'action-link/macro.njk' import actionLink %}
+
+{% set href="https://github.com/nhsbsa/nhsbsa-digital-playbook/discussions/new?&template=" +
```

Comment:
This URL is wrong. We would need it to be: https://github.com/nhsbsa/nhsbsa-digital-playbook/discussions/new?category=general-playbook-ideas 

Thats if we're directing users to general-playbook-ideas? If not then we could make it so that the discussion category is customisable?